.github/workflows/schedule.yml

```diff
@@ -127,7 +127,7 @@ jobs:
           context: .
           file: ./Dockerfile.ui
           cache-from: type=local,src=/tmp/cvat_cache_ui
-          tags: cvat/ui:latest
+          tags: cvat/ui:dev
```

Comment:
Is this image actually used for anything? It seems like the UI doesn't actually get started at any point in this job.

And if it _is_ needed, wouldn't it make more sense to pull the image that's already been built, instead of building it anew?